jquery/jquery.js

```diff
@@ -0,0 +1,342 @@
+
+$(document).ready(function() {
+
+  //       document.getElementsByName('frame1roll1')[0].onchange = function() {
+  //      if (this.value=="blank") alert('Select something !');
+  // }
+
+
+  $("#bonus-1-title").hide();
+  $("#bonus-roll-1").hide();
+  $("#bonus-roll-2").hide();
+  $("#bonus-add").hide();
+  $(".bonustwo").hide();
+
+
+  $("#frame-10-add").click(function() {
+    val1 = $("#frame-10-roll-1").val();
+    val1number = val1*1
+    val2 = $("#frame-10-roll-2").val();
+    val2number = val2*1
+
+    if(val1number == 10) {
+      $("#bonus-1-title").show();
+      $("#bonus-roll-1").show();
+      $("#bonus-roll-2").show();
+      $("#bonus-add").show();
+    } else if ((val1number + val2number == 10) && val1number != 10) {
+      $("#bonus-1-title").show();
+      $("#bonus-roll-1").show();
+      $("#bonus-add").show();
+    } else {
+      $(".final-score").css("display", "inline");
+      $(".score").css("display", "none");
+    }
+  })
+
+  $("#bonus-add").click(function() {
+    val = $("#bonus-roll-1").val();
+    if(val == 10) {
+      $(".bonustwo").show();
+    }
+    else {
+      $(".final-score").css("display", "inline");
+      $(".score").css("display", "none");
+    }
+  });
+
+
+
+  $("#reset").on("click", function () {
+    $("#frame-1-roll-1").val("");
+    $("#frame-1-roll-2").val("");
+    $("#frame-1-add").css("display", "inline");
+    $("#frame-2-roll-1").val("");
+    $("#frame-2-roll-2").val("");
+    $("#frame-2-add").css("display", "inline");
+    $("#frame-3-roll-1").val("");
+    $("#frame-3-roll-2").val("");
+    $("#frame-3-add").css("display", "inline");
+    $("#frame-4-roll-1").val("");
+    $("#frame-4-roll-2").val("");
+    $("#frame-4-add").css("display", "inline");
+    $("#frame-4-roll-1").val("");
+    $("#frame-4-roll-2").val("");
+    $("#frame-4-add").css("display", "inline");
+    $("#frame-5-roll-1").val("");
+    $("#frame-5-roll-2").val("");
+    $("#frame-5-add").css("display", "inline");
+    $("#frame-6-roll-1").val("");
+    $("#frame-6-roll-2").val("");
+    $("#frame-6-add").css("display", "inline");
+    $("#frame-7-roll-1").val("");
+    $("#frame-7-roll-2").val("");
+    $("#frame-7-add").css("display", "inline");
+    $("#frame-8-roll-1").val("");
+    $("#frame-8-roll-2").val("");
+    $("#frame-8-add").css("display", "inline");
+    $("#frame-9-roll-1").val("");
+    $("#frame-9-roll-2").val("");
+    $("#frame-9-add").css("display", "inline");
+    $("#frame-10-roll-1").val("");
+    $("#frame-10-roll-2").val("");
+    $("#frame-10-add").css("display", "inline");
+    $("#bonus-1-title").hide();
+    $("#bonus-roll-1").hide();
+    $("#bonus-roll-2").hide();
+    $("#bonus-add").hide();
+    $(".bonustwo").hide();
+    $(".final-score").css("display", "none");
+    $(".score").css("display", "inline");
+
```

Comment:
Could maybe think about using an iterator here to dry it up a bit.  Bit like this.

for (var i = 1; i < 11; i++) {
  $("#frame-"+ i +"-roll-1").val("");
    $("#frame-"+ i +"-roll-2").val("");
    $("#frame-"+ i +"-add").css("display", "inline");
}